language/cc/resolve.go

```diff
@@ -222,29 +247,32 @@ func compareLabels(l, r label.Label) int {
 	return strings.Compare(l.String(), r.String())
 }
 
-// Tries to resolve given importSpec using the following strategies:
+var (
+	errResolveErrorUnresolved              = errors.New("could not resolve the given importSpec")
+	errResolveErrorSelfImport              = errors.New("importSpec was resolved as a self-import")
+	errResolveErrorMissingModuleDependency = errors.New("importSpec was resolved to an external dependency unmentioned in MODULE.bazel")
```

Comment:
`header file found in external library not declared in MODULE.bazel`.

This should really say what the module is though.